### PR TITLE
Tablegrid.filter.accent.insensitive

### PR DESCRIPTION
**Pull request by GuillaumeNury**
- `tablegrid` filter is now acent insensitive (it was only case insensitive)

### CODE REVIEW

**Comment by Poltuu** [comment]
this should probably be a directive

**Comment by GuillaumeNury** [comment]
I use this function as a filter in CC.
I don't know if such a filter has a place in LuccaUI as it not related to UI.

**Comment by Poltuu** [comment]
to me, that's a much needed filter pretty much in every search input

**Comment by GuillaumeNury** [comment]
here is... luifStripAccent !

**Comment by Poltuu** [comment]
mais du coup, autant copier ça non ? 
https://github.com/Kepro/angular-remove-diacritics/blob/master/remove-diacritics.js
ou ça, mais c moins puissant
https://gist.github.com/monkeymonk/ccf698e7b71ba22f098a

@lucienbertin

File: js/filters/genericFilters.js [hunk 1]
```diff
@@ -59,5 +59,33 @@
 			var integerPart = text.split(separator)[0];
 			return $sce.trustAsHtml(integerPart + rightSpan);
 		};
-	}]);
-})();
\ No newline at end of file
+	}])
```
Comment (lucienbertin) [inline js/filters/genericFilters.js]: could you add it to the typescript section of the code, also also add some unit test

File: js/lui.js [hunk 2]
```diff
@@ -17,7 +17,7 @@
 	// all the translations in one module
 	angular.module('lui.translates.userpicker', []);
 	angular.module('lui.translates.daterangepicker', []);
-	angular.module('lui.translates.tablegrid', []);
+	angular.module('lui.translates.tablegrid', ['lui.filters']);
```
Comment (lucienbertin) [inline js/lui.js]: ???

the module `'ui.translates.tablegrid` only contains the translations of the tablegrid, it has no reason to depend on `lui.filters`

**Comment by GuillaumeNury** [comment]
do we really need the full map ?

**Comment by Poltuu** [comment]
but do we really need half a map that's gonna have to be updated in xx months ?

**Comment by GuillaumeNury** [comment]
Tests added + full diacritics map + LINT

**Pull request merged**